mcs/class/Mono.Posix/Mono.Unix.Native/Stdlib.cs

```diff
@@ -378,7 +380,11 @@ static XPrintfFunctions ()
 	//
 	public class Stdlib
 	{
+#if NETSTANDARD2_0
+		internal const string LIBC = "c";
```

Comment:
I decided to go the FORCE_USE_LIBC_NOT_MSVC route.  It allows me to build a special version of a netstandard2.0 library that will use "c".  This will allow us to switch in this instance at runtime using NuGet.